DIPs/DIP-55.md

```diff
@@ -0,0 +1,45 @@
+---
+DIP: 55
+Title: Security Awareness Activities On-Site  
+Status: Draft  
+Themes: Community Involvement, Social  
+Tags: awareness, campaign, CTF, phishing, red-team, security 
+Instances: *  
```

Comment:
This won't happen at devcon7, I don't know which one yet. We won't be able to do all of this with all our current obligations, hence the *. Because if this is something the community likes, we can do it at other conference editions.

Let me know how you would like to proceed to know this (also clarified in out forum post as well)

Reply:
unfortunately we cannot do * as we cannot travel to the past and can't guarantee the future ;-)
We can only target specific events. Maybe we target it DEVCon8 then?

Reply:
Oh hahah, it was suggested in the documentation: `DIP-0.md`.
Yeah, that seems fine (:

Reply:
I'll add the modification, and squash

Reply:
Oh - fixed: https://github.com/efdevcon/DIPs/pull/164